gwswlib/exporter.py

```diff
@@ -220,53 +223,51 @@ def write_threedi_to_db(threedi, threedi_db_settings):
         del pump["start_node.code"]
         del pump["end_node.code"]
 
-        obj_list.append(Pumpstation(**pump))
+        pump_list.append(Pumpstation(**pump))
 
-    commit_counts["pumpstations"] = len(obj_list)
-    session.bulk_save_objects(obj_list)
+    commit_counts["pumpstations"] = len(pump_list)
+    session.bulk_save_objects(pump_list)
     session.commit()
-    del obj_list
 
-    # for weir in threedi.weirs:
-    #     try:
-    #         weir['connection_node_start_id'] = con_dict[
-    #             weir['start_node.code']]
-    #     except KeyError:
-    #         self.log.add(
-    #             logging.ERROR,
-    #             'Start node of weir not found in nodes',
-    #             {},
-    #             'Start node {start_node} of weir with code {code} not '
-    #             'found',
-    #             {'start_node': weir['start_node.code'],
-    #                 'code': weir['code']}
-    #         )
+    weir_list = []
+    for weir in threedi.weirs:
+        if weir["start_node.code"] in connection_node_dict:
+            weir["connection_node_start_id"] = connection_node_dict[
+                weir["start_node.code"]
+            ]
+        else:
+            weir["connection_node_start_id"] = None
+            logger.error(
+                "Start node of weir %r not found in connection nodes", weir["code"]
+            )
 
-    #     try:
-    #         weir['connection_node_end_id'] = con_dict[
-    #             weir['end_node.code']]
-    #     except KeyError:
-    #         self.log.add(
-    #             logging.ERROR,
-    #             'End node of weir not found in nodes',
-    #             {},
-    #             'End node {end_node} of weir with code {code} not found',
-    #             {'end_node': weir['end_node.code'], 'code': weir['code']}
-    #         )
+        if weir["end_node.code"] in connection_node_dict:
+            weir["connection_node_end_id"] = connection_node_dict[weir["end_node.code"]]
+        else:
+            weir["connection_node_end_id"] = None
+            logging.error(
+                "End node of weir %r not found in connection nodes", weir["code"]
+            )
+
+        weir["cross_section_definition_id"] = cross_section_dict[
+            weir["cross_section_code"]
+        ]
 
-    #     weir['cross_section_definition_id'] = crs_dict[weir['crs_code']]
+        del weir["start_node.code"]
```

Comment:
Dit is een ander item dan het andere PR, toch?
Ook hier misschien even kijken of het goed gaat (of nodig is) met die `del`.